django_reference_app/tests/test_homepage.py

```diff
@@ -0,0 +1,6 @@
+from django import test
+
```

Comment:
The Python style guide, PEP8, dictates that there should be [two blank lines before a class definition](https://www.python.org/dev/peps/pep-0008/#blank-lines).